spec/computer_spec.rb

```diff
@@ -1,8 +1,32 @@
 require "computer"
+require "spec_helper"
+require "board"
 
 RSpec.describe Computer do
-  it "creates a new computer with a mark" do
-    computer = Computer.new("O")
+  board = Board.new(empty_board)
+  computer = Computer.new("O", board)
+
+  it "creates a new computer player with a mark" do
     expect(computer.mark).to eq("O")
   end
+  
+  it "selects an available move from the board" do
+    expect(computer.get_move.nil?).to eq(false)
+  end
+
+  it "is able to work out it's opponent's mark" do
+    expect(computer.opponent_mark).to eq("X")
```

Comment:
I'd love to see the setup here where you pass "O" into the player, having those two things next to each other make it much more readable in isolation which is something I value heavily.

Reply:
Method removed, opponent mark is now only discovered locally within the block_opponent method